src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((prev, singleStyle) => {
+    if (!singleStyle.trim()) {
+      return prev;
+    }
+
+    const [property, value] = singleStyle.split(':');
```

Comment:
Consider handling cases where `singleStyle.split(':')` results in an array with more than two elements. This can happen if the value part contains colons, which might be valid in some contexts.